### PR TITLE
feat: Add `REMOTE_FILE`

### PR DESCRIPTION
**Pull request by jjerphan**
### Description

- [x] I've included a concise description of what each node does

### Styleguide

- [x] My node adheres to the [styleguide](https://github.com/flojoy-io/nodes/blob/main/node_styleguide.md) for Flojoy nodes

### Docs

- [x] I've submitted a PR for a documentation page for the new node(s) that contains usage examples (see docs.flojoy.io)

### Testing

- [x] This PR includes a unit test ([example here](https://github.com/flojoy-io/nodes/blob/c28808f2d0aec8f116cdb3fccb46c4f5256574e9/TRANSFORMERS/ARITHMETIC/ADD/ADD_test_.py) and/or ideally a screenshot of the node's output on an example app.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -47,6 +47,6 @@ xlrd==2.0.1
 pytest==7.4.0
 pytest-test-groups==1.0.3
 pytest-xdist==3.3.1
-flojoy==0.1.10
+flojoy==0.1.11
```
Comment (jjerphan) [inline requirements.txt:50]: Note for reviewers: this is required for proper integration within `studio` to use `flojoy.display` (tests are failing on `main` for this reason currently).

File: LOADERS/REMOTE_FILE_SYSTEM/REMOTE_FILE/REMOTE_FILE.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+def get_file_url(url: str):
+    parse_result = urlparse(url)
+    if not all([parse_result.scheme, parse_result.netloc]):
+        raise ValueError(f"'{url}' is not a valid URL.")
+    return url
+
+
+@flojoy(
+    deps={
```
Comment (Roulbac) [inline LOADERS/REMOTE_FILE_SYSTEM/REMOTE_FILE/REMOTE_FILE.py:21]: Adding a TODO for myself to replace all `@flojoy(deps={...})` with  `run_in_venv` in a single PR + decide what goes in the main env

File: LOADERS/REMOTE_FILE_SYSTEM/REMOTE_FILE/REMOTE_FILE.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        "openpyxl": "3.0.10",
+        "scikit-image": "0.21.0",
+    }
+)
+def REMOTE_FILE(
+    file_url: str = None,
+    default: Optional[TextBlob] = None,
+    file_type: Literal["Image", "Grayscale", "JSON", "CSV", "Excel", "XML"] = "Image",
+) -> Image | DataFrame:
+    """The REMOTE_FILE node loads a remote file of a different type and converts it to a DataContainer class.
```
Comment (Roulbac) [inline LOADERS/REMOTE_FILE_SYSTEM/REMOTE_FILE/REMOTE_FILE.py:33]: I would specify that this only supports HTTP file URLs and that `gcp://`, `s3://` and other bucket-like URIs are not supported

**Comment by itsjoeoui** [comment]
Hey @jjerphan ! Looks good, one last thing, can you add an example app (`app.json`) and a example.md like other nodes. After that I will go ahead and merge this.

File: LOADERS/REMOTE_FILE_SYSTEM/REMOTE_FILE/REMOTE_FILE.py [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+@flojoy(
+    deps={
+        "xlrd": "2.0.1",
+        "lxml": "4.9.2",
+        "openpyxl": "3.0.10",
+        "scikit-image": "0.21.0",
+    }
+)
+def REMOTE_FILE(
```
Comment (Roulbac) [inline LOADERS/REMOTE_FILE_SYSTEM/REMOTE_FILE/REMOTE_FILE.py:28]:
```suggestion
# TODO: Consolidate remote files (S3, HTTP, etc...) and perhaps even local files in a single node
def REMOTE_FILE(
```

**Review by Roulbac — APPROVED** [review]
LGTM, thank you @jjerphan !

**Review by Roulbac — APPROVED** [review]
Thank you @jjerphan

**Pull request merged**